src/content/docs/mobile-monitoring/new-relic-mobile-android/install-configure/install-android-apps-gradle-android-studio.mdx

```diff
@@ -33,115 +33,188 @@ redirects:
   - /docs/mobile-monitoring/new-relic-mobile-android/legacy/installing-android-apps-eclipse
   - /docs/mobile-monitoring/new-relic-mobile-android/install-configure/installing-android-apps-maven
   - /docs/mobile-monitoring/new-relic-mobile-android/install-configure
+  - /docs/mobile-monitoring/new-relic-mobile-android/install-configure/configure-android-gradle-android-studio
 ---
 
-
-We recommend that you use our [guided install](https://onenr.io/0kjnEoryzRo) to set up Android monitoring. However, if you need to install the agent manually, follow the steps below to install the New Relic Android agent with Gradle and Android Studio.
-
-<InlineSignup/>
+You can install the mobile agent for Android monitoring with our [guided install](https://onenr.io/0kjnEoryzRo), or you can install manually. The guided install is a good option for simple app environments. For other cases, you can follow the steps below to install the Android agent with Gradle and Android Studio.
 
 ## Install your Android app [#installation]
 
 As part of the installation process for mobile monitoring, New Relic automatically generates an [application token](/docs/mobile-apps/viewing-your-application-token). This is a 40-character hexadecimal string for authenticating each mobile app you monitor in New Relic.
 
+<Steps>
+    <Step>
+        ### Add your Android app to New Relic
+
 1. Go to **[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > Mobile**.
-2. If applicable: From the [**Mobile apps** index](/docs/mobile-monitoring/mobile-monitoring-ui/mobile-app-pages/mobile-apps-index), select **Add a new app**.
-3. From the **Get started** page, select **Android** as the platform for mobile monitoring.
+2. From the [**Mobile apps** index](/docs/mobile-monitoring/mobile-monitoring-ui/mobile-app-pages/mobile-apps-index), select **Add a new app**.
+3. From the **Get started** page, select **Android** as your mobile monitoring platform.
 4. Enter a name for your mobile app, and select **Continue**.
 
-Continue with the steps to [configure mobile monitoring](#configuration) with Gradle and Android Studio.
+    </Step>
+    <Step>
+        ### Update your Gradle build file on the project and/or app level
```

Comment:
@cthomas-newrelic Based on the configuration doc, it seems like there are two options. You can make general configurations on the project level, which affects all apps within that project. You can make further, perhaps more specific configurations on the app levels. 

- Is this accurrate?
- How do customers know if they should do one, the other, or both?